google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/RowMutationEntry.java

```diff
@@ -177,4 +178,11 @@ public RowMutationEntry deleteRow() {
         .addAllMutations(mutation.getMutations())
         .build();
   }
+
+  /** Wraps the protobuf {@link MutateRowRequest}. */
+  public static RowMutationEntry fromProtobuf(MutateRowsRequest.Entry request) {
+    // TODO: Should we have an option for Mutation.fromProto(List<com.google.bigtable.v2.Mutation>)?
+    return new RowMutationEntry(
+        request.getRowKey(), Mutation.fromProtoUnsafe(request.getMutationsList()));
+  }
```

Comment:
I dont think we need this...its only used by BulkMutation and if someone is using BulkMutation, they are already using the models